hack/fuse-demo/wrap_datamon.sh

```diff
@@ -245,11 +266,22 @@ ifs_orig="$IFS"
 IFS=':'
 for upload_cmd in $UPLOAD_CMDS; do
     IFS="$ifs_orig"
-    if ! run_datamon_cmd "$upload_cmd"; then
-        upload_status="$?"
+    bundle_id=$(2>&1 run_datamon_cmd "$upload_cmd" | \
+                    grep 'Uploaded bundle id' | \
+                    head -1 | \
+                    sed 's/Uploaded bundle id:\(.*\)/\1/')
```

Comment:
hey so looks like the output of `run_datamon_cmd` looks like this: 
```
'{"level":"info","ts":1565158923.6535618,"caller":"core/bundle_pack.go:307","msg":"Uploaded
            bundle id","BundleID":"1P5LHRMo8EipZGg5FJW193e6SkS"}'
```
so the sed command will have to change a bit

Reply:
there are, conceptually, at least two bytestreams getting muxed into stdout/stderr:  the `zap` logger ( https://github.com/oneconcern/datamon/blob/master/pkg/core/bundle_pack.go#L307 ) and the stdlib `log` logger ( https://github.com/oneconcern/datamon/blob/master/cmd/datamon/cmd/bundle_upload.go#L85 ).

the stdlib logger, the one that prints `Uploaded bundle id:`, is the bytestream used in test ( https://github.com/oneconcern/datamon/blob/master/cmd/datamon/cmd/cli_test.go#L335 ), so that's the bytestream the regexp is written for.

Reply:
Got it, I think that makes sense.
Is there a way to pass the bytstream from the stdlib logger instead of the zap logger to the regex? At the moment the text being written to the 'bundle_id' file (use as an argo output parameter) is the same as the zap logger output